src/components/IssuePr/Description.tsx

```diff
@@ -0,0 +1,128 @@
+import { Menu, Transition } from '@headlessui/react'
+import MDEditor from '@uiw/react-md-editor'
+import { formatDistanceToNow } from 'date-fns'
+import React, { Fragment, useEffect } from 'react'
+
+import { Button } from '@/components/common/buttons'
+import { shortenAddress } from '@/helpers/shortenAddress'
+import { useGlobalStore } from '@/stores/globalStore'
+import { Issue, PullRequest } from '@/types/repository'
+
+export default function Description({ issueOrPr }: { issueOrPr: Issue | PullRequest }) {
+  const [isEditing, setIsEditing] = React.useState(false)
+  const [isSubmittingDescription, setIsSubmittingDescription] = React.useState(false)
+  const [description, setDescription] = React.useState('')
+  const [isContributor, updateIssueDetails, updatePullRequestDetails] = useGlobalStore((state) => [
+    state.repoCoreActions.isContributor,
+    state.issuesActions.updateIssueDetails,
+    state.pullRequestActions.updatePullRequestDetails
+  ])
+  const contributor = isContributor()
+  const isIssue = 'assignees' in issueOrPr
+
+  async function handleUpdateDescription() {
+    if (issueOrPr) {
+      setIsSubmittingDescription(true)
+
+      if (issueOrPr.description !== description) {
+        if (isIssue) {
+          await updateIssueDetails(issueOrPr?.id, { description })
+        } else {
+          await updatePullRequestDetails(issueOrPr?.id, { description })
+        }
+      }
+
+      setIsSubmittingDescription(false)
+      setIsEditing(false)
+    }
+  }
+
+  function handleOnCancelDescription() {
+    setIsEditing(false)
+  }
+
+  useEffect(() => {
+    if (issueOrPr?.description) {
+      setDescription(issueOrPr.description)
+    }
+  }, [issueOrPr?.description])
+
+  if (!issueOrPr) return null
+
+  return (
+    <div className="flex flex-col border-gray-300 border-[1px] w-full rounded-lg bg-white overflow-hidden">
+      <div
+        className={`flex justify-between items-center gap-3 bg-gray-200 text-gray-900 px-4 ${
+          isIssue ? 'py-2' : 'py-[1.5px]'
+        } border-b-[1px] border-gray-300`}
+      >
+        {isIssue ? (
+          <div className="flex flex-auto justify-between">
+            <span>{shortenAddress(issueOrPr.author)}</span>
+            <span> {formatDistanceToNow(new Date(issueOrPr.timestamp), { addSuffix: true })}</span>
+          </div>
+        ) : (
+          <div className="font-medium">Description</div>
```

Comment:
@pawanpaudel93 we lost the previous styles here when this new component replaced in PR read overview section

Reply:
It's resolved.